src/contracts/test/GPv2SettlementTestInterface.sol

```diff
@@ -1,10 +1,19 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
 pragma solidity ^0.6.12;
-pragma experimental ABIEncoderV2;
 
 import "../GPv2Settlement.sol";
 
 contract GPv2SettlementTestInterface is GPv2Settlement {
+    constructor(GPv2AccessControl _controller)
+        public
+        GPv2Settlement(_controller)
+    // solhint-disable-next-line no-empty-blocks
+    {
+        // According to the solidity docs:
+        // https://docs.soliditylang.org/en/develop/contracts.html#arguments-for-base-constructors
+        // This is how we inherit base constructors.
```

Comment:
I don't think this comment is necessary.

Reply:
I agree although I found it quite unnatural to have to even include the brackets. I would think there could be some nicer way of inheriting the super constructor.